source/features/clean-repo-tabs.tsx

```diff
@@ -40,6 +40,22 @@ const getWikiPageCount = cache.function(async (): Promise<number> => {
 	cacheKey: () => 'wiki-page-count:' + getRepo()!.nameWithOwner,
 });
 
+const getWorkflowsCount = cache.function(async (): Promise<number> => {
+	const {repository: {workflowFiles}} = await api.v4(`
+		repository() {
+			workflowFiles: object(expression: "HEAD:.github/workflows") {
+				... on Tree { entries { oid } }
```

Comment:
Seems weird we have to fetch something just to count the entries, there has to be a better way :thinking:

Reply:
I assume it's
```suggestion
				totalCount
```

Reply:
I thought so but it doesn't work. The docs don't mention any `length` or `count` fields on `Tree` or `entries`, so I think we're stuck with this :grimacing: